tests/test_dfa.py

```diff
@@ -1396,3 +1397,46 @@ def test_show_diagram_write_file(self):
         self.dfa.show_diagram(path=diagram_path)
         self.assertTrue(os.path.exists(diagram_path))
         os.remove(diagram_path)
+
+
+    def test_minimal_finite_language(self):
+        """Should compute the minimal DFA accepting the given finite language"""
+
+        # Same language described in the book this algorithm comes from
+        language = ['aa','aaa','aaba','aabbb','abaa','ababb','abbab','baa','babb','bbaa','bbabb','bbbab']
```

Comment:
@eliotwrobson Shouldn't this be a `set`, not a `list`?

Reply:
Good catch!